packages/sources/jpegd/src/config.ts

```diff
@@ -0,0 +1,14 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
+export const NAME = 'JPEGD'
+
+export const DEFAULT_ENDPOINT = 'punks'
+export const DEFAULT_BASE_URL = 'https://jpegapi.com'
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix)
```

Comment:
If `API_KEY` is required, this line should be changed to:
```suggestion
  const config = Requester.getDefaultConfig(prefix, true)
```

Reply:
I believe this will fix the failing unit test too

Reply:
Changed